nin/backend/watch.js

```diff
@@ -29,12 +29,21 @@ function watch(projectPath, cb) {
    * trigger intial callbacks for all files */
   watcher.on('add', function(){ });
 
+  var graphAlreadyLoaded = false;
+
   watcher.on('all', function (event, path) {
     if (event === 'unlink') event = 'delete';
     if (event == 'addDir') {
       return;
     }
 
+    if(path == 'res/graph.json') {
```

Comment:
Should we add an issue explaining this change, and linking to it from the code?
It's mentioned in the PR (that editing mainly will happen from UI) but its bound to bite someone in the ass at one point when they can't figure out why their graph.json isn't pushed over the intertubes.